src/coreclr/vm/interpreter.h

```diff
@@ -537,7 +537,17 @@ struct CachedItem
     }
 };
 
+// Also see namedintrinsiclist.h
+enum InterpreterNamedIntrinsics : unsigned short
+{
+    NI_Illegal = 0,
+    NI_System_StubHelpers_GetStubContext,
+    NI_System_Runtime_InteropService_MemoryMarshal_GetArrayDataReference,
+};
+
+InterpreterNamedIntrinsics getNamedIntrinsicID(CEEInfo* info, CORINFO_METHOD_HANDLE methodHnd);
 
+const char* getMethodName(CEEInfo* info, CORINFO_METHOD_HANDLE hnd, const char** className, const char** namespaceName = NULL, const char **enclosingClassName = NULL);
```

Comment:
Can these methods be defined as `static` within the `Interpreter` type or within the .cpp file so that they do not pollute the global scope?